apps/backend/src/evaluations/dto/create-evaluation.dto.ts

```diff
@@ -1,26 +1,16 @@
 import {ICreateEvaluation} from '@heimdall/interfaces';
-import {
-  IsArray,
-  IsBoolean,
-  IsNotEmpty,
-  IsObject,
-  IsOptional,
-  IsString
-} from 'class-validator';
+import {IsArray, IsNotEmpty, IsOptional, IsString} from 'class-validator';
 import {CreateEvaluationTagDto} from '../../evaluation-tags/dto/create-evaluation-tag.dto';
 
 export class CreateEvaluationDto implements ICreateEvaluation {
   @IsNotEmpty()
   @IsString()
   readonly filename!: string;
 
-  @IsNotEmpty()
-  @IsObject()
   readonly data!: Record<string, any>;
 
   @IsNotEmpty()
-  @IsBoolean()
-  readonly public!: boolean;
+  public!: boolean;
```

Comment:
Shouldn't `IsBoolean` still be present here?

Reply:
It doesn't work with IsBoolean, it still returns an API error if the field is a string.